src/matlab.ts

```diff
@@ -46,27 +39,39 @@ export async function setupBatch(platform: string) {
 }
 
 export async function getReleaseInfo(release: string): Promise<Release> {
-    const client: http.HttpClient = new http.HttpClient();
-    const releaseInfo = await client.getJson<MATLABReleaseInfo>(properties.matlabReleaseInfoUrl);
-
-    if (!releaseInfo.result) {
-        return Promise.reject(Error(`Unable to retrieve the MATLAB release information. Contact MathWorks at continuous-integration@mathworks.com if the problem persists.`));
+    // Get release name from input parameter
+    let name: string;
+    if ( release.toLowerCase().trim() === "latest") {
+        try {
+            const client: http.HttpClient = new http.HttpClient();
+            const latestResp = await client.get(properties.matlabLatestReleaseUrl);
+            name = await latestResp.readBody();    
+        }
+        catch {
+            return Promise.reject(Error(`Unable to retrieve the MATLAB release information. Contact MathWorks at continuous-integration@mathworks.com if the problem persists.`));
+        }
+    } else {
+        let nameMatch = release.toLowerCase().match(/r[0-9]{4}[a-b]/);
```

Comment:
We aren't allowing users to specify update level?

Reply:
We are, we just pull the release name and the update level out separately

Reply:
Updated to warn if formatting is invalid. Using latest as default in this case